src/shared/components/table-header/table-header.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from './table-header.module.scss';
+
+export const TableHeader = () => {
+  return (
+    <tr>
```

Comment:
IMHO I think headers should be an array that is passed to this component and it maps the array and renders it accordingly. that would make this component truly reusable